docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: "3.9"
+
+services:
+  app:
+    build:
+      args:
+        user: opendor-dev
+        uid: 1000
+      context: ./
+      dockerfile: .docker/php/Dockerfile
+    image: php:8.0
```

Comment:
The dockerfile says `FROM php:8.0-fpm` but here you also define another `image`. 🤔
Isn't that duplicated and conflicting?

Reply:
Removed, nice catch